### PR TITLE
ad9172: Add builds.json, fix project and update to iio_app

### PR DESCRIPTION
**Pull request by mchindri**
Not tested

### CODE REVIEW

File: projects/ad9172/builds.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            },
+            "dma_example": {
+                  "flags": "NEW_CFLAGS=-DDAC_DMA_EXAMPLE",
+                  "hardware": [
+                        "dac_fmc_ebz_zc706",
+                        "dac_fmc_ebz_zcu102"
+                  ]
+            }
+      }
+}
```
Comment (amiclaus) [inline projects/ad9172/builds.json:25]: nit: new line 😄

**Review by amiclaus — APPROVED** [review]

**Comment by mchindri** [comment]
v2:
 - Add new line

**Comment by mchindri** [comment]
v3:
 - rebase onto latest master fix

**Comment by mchindri** [comment]
v4: 
 - Fix conflicts

**Comment by mchindri** [comment]
Tested

**Pull request merged**